spec/controllers/formats_controller_spec.rb

```diff
@@ -27,7 +16,8 @@
 
     it "returns supported formats" do
       json_response = JSON.parse response.body
-      expect(json_response['export_formats'].first).to have_key 'format'
+      expect(json_response['export_formats']).to be_an Array
+      expect(json_response['export_formats']).to be_present
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.